fdbserver/storageserver.actor.cpp

```diff
@@ -697,6 +697,19 @@ struct StorageServer {
 	Reference<Histogram> storageCommitLatencyHistogram;
 	Reference<Histogram> ssDurableVersionUpdateLatencyHistogram;
 
+	struct PhysicalShardHistograms {
```

Comment:
Will TraceEvent be sufficient for physical shard cleanup? Histogram feels an overkill.